spring-file/jmx-run.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/sh
+# Run performance tests for Spring Boot server locally
+
```

Comment:
I'd suggest adding `set -eo pipefail` for any script:
 - `-e` for "Exit immediately if a command exits with a non-zero status."
 - `-o pipefail` for "the return value of a pipeline is the status of the last command to exit with a non-zero status, or zero if no command exited with a non-zero status"